src/main/java/seedu/address/model/person/Interaction.java

```diff
@@ -38,6 +42,7 @@ public static boolean isValidOutcome(String test) {
     public Interaction(String interactionNote) {
         this.interactionNote = interactionNote;
         this.outcome = Outcome.UNKNOWN;
+        this.date = new Date();
     }
```

Comment:
Feel that the lead should be a compulsory field for each interaction, dont think there is a need for an unknown outcome

Reply:
Its easier for the user this way, imagine writing a whole list of text only to have it be rejected because you didn't do the o/OUTCOME properly